test/kafka-multi-broker/mzcompose.yml

```diff
@@ -1,126 +0,0 @@
-# Copyright Materialize, Inc. and contributors. All rights reserved.
-#
-# Use of this software is governed by the Business Source License
-# included in the LICENSE file at the root of this repository.
-#
-# As of the Change Date specified in that file, in accordance with
-# the Business Source License, use of this software will be governed
-# by the Apache License, Version 2.0.
-
-version: "3.7"
-
-mzworkflows:
-  kafka-multi-broker:
-    steps:
-      - step: start-services
-        services: [kafka1, kafka2, kafka3]
-
-      - step: start-services
-        services: [materialized]
-
-      - step: run
-        service: testdrive-svc
-        command: --kafka-addr=kafka2 01-init.td
-
-      - step: sleep
-        duration: 10
-
-      - step: kill-services
-        services: [kafka1]
-        signal: SIGKILL
-
-      - step: sleep
-        duration: 10
-
-      - step: run
-        service: testdrive-svc
-        command: --kafka-addr=kafka2,kafka3 --no-reset 02-after-leave.td
-
-      - step: start-services
-        services: [kafka1]
-
-      - step: sleep
-        duration: 10
-
-      - step: run
-        service: testdrive-svc
-        command: --kafka-addr=kafka1 --no-reset 03-after-join.td
-
-services:
-  testdrive-svc:
-    mzbuild: testdrive
-    entrypoint:
-      - bash
-      - -c
-      - >-
-        wait-for-it --timeout=30 kafka2:9092 &&
-        wait-for-it --timeout=30 kafka3:9092 &&
-        wait-for-it --timeout=30 materialized:6875 &&
-        wait-for-it --timeout=30 schema-registry:8081 &&
-        testdrive
-        --seed=1
-        --kafka-option acks=all
-        --default-timeout=300s
-        --schema-registry-url=http://schema-registry:8081
-        --materialized-url=postgres://materialize@materialized:6875
-        $$*
-      - bash
-    volumes:
-      - .:/workdir
-    init: true
-    depends_on: [kafka2, schema-registry, materialized]
-
-  materialized:
-    mzbuild: materialized
-    command: -w1 --disable-telemetry
-    environment:
-      - MZ_DEV=1
-
-  zookeeper:
-    image: confluentinc/cp-zookeeper:5.5.4
-    environment:
-      - ZOOKEEPER_CLIENT_PORT=2181
-      - ZOO_TICK_TIME=500
-
-  schema-registry:
-    image: confluentinc/cp-schema-registry:5.5.4
-    environment:
-    - SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS=PLAINTEXT://kafka2:9092
-    - SCHEMA_REGISTRY_HOST_NAME=localhost
-    depends_on: [kafka2, zookeeper]
-
-  kafka1:
-    image: confluentinc/cp-kafka:5.5.4
-    hostname: kafka1
-    environment:
-      - KAFKA_BROKER_ID=1
-      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181
-      - KAFKA_ADVERTISED_HOST_NAME=kafka1
-      - KAFKA_ADVERTISED_LISTENERS=PLAINTEXT://kafka1:9092
-      - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=2
```

Comment:
I think `KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=2` needs to remain in the python-based version of the test. Otherwise killing kafka1 may leave the kafka cluster without a node that has the offsets topic

Reply:
Good point! Fixed.